misctools/mime.js

```diff
@@ -21,7 +21,7 @@ const withExtensions = [
         return !!json[key]?.extensions?.length;
       })
       .flatMap(mime => {
-        return [...new Set([...json[mime].extensions])].map(ext => {
```

Comment:
This is to `_.uniq` the values of `json[mime].extensions` (not present in all datasets but filtered above). It looks like the dataset from `https://raw.githubusercontent.com/jshttp/mime-db/master/db.json` does not contain duplicates anyways so the entire thing could be simplified to:
```js
const withExtensions = [];
{
  const mimes = Object.keys(json);
  for (let i = 0, { length: mimesLen } = mimes; i < mimesLen; i += 1) {
    const mime = mimes[i];
    const exts = json[mime]?.extensions;
    for (let j = 0, extsLen = exts?.length || 0; j < extsLen; j += 1) {
      withExtensions.push(`.{.@"${exts[j]}", all.@"${mime}"}`);
    }
  }
}
```
Which is 11x faster too.
<img width="195" alt="11x faster" src="https://github.com/oven-sh/bun/assets/4303/8501f11c-b570-4f61-939d-6603f24a3d51">

Reply:
true
I was just removing the unnecessary spread as I just found it annoying; clearly I keep the uniqueness and sorting assuming they were useful.
If uniqueness and sorting are not necessary we could always:
```js
const withExtensions = [];
const mimes = Object.keys(json);
for (let i = 0; i < mimes.length; i++) {
    const mime = mimes[i];
    const { extensions } = json[mime] || { extensions: [] };
    if (extensions) {
        const formattedExtensions = extensions.map(ext => `.{.@"${ext}", all.@"${mime}"}`);
        withExtensions.push(...formattedExtensions);
    }
}
```
🤷🏼 Not sure the quality of JSBench.me benchmarking (any bun bench marking utilities you recommend?):
<img width="1001" alt="image" src="https://github.com/oven-sh/bun/assets/4691147/9857624c-407e-4fb9-b8e3-868973c65e6c">

Reply:
_RE the screenshot edited out: Microbenmarks get grumpy about closures_